tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "ES2019",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "esModuleInterop": false,
+    "noEmitOnError": true,
+    "lib": ["esnext", "dom", "dom.iterable"],
+    "strict": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
this one seems confusing IMHO. In [this example](https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports), I would expect it to import a `React` object from "react", not import the defaults.

Reply:
No preference, I've removed it for now. Some packages (used to) depend on it.